spec/rails_helper.rb

```diff
@@ -13,10 +13,13 @@ module Features
 end
 
 RSpec.configure do |config|
+  config.before(:suite) do
+    Rails.application.load_seed
```

Comment:
@edwardloveall FYI I also needed to copy this line from the now-deleted `database_cleaner.rb`.